backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/predicate/impl/LuceneSimpleQueryStringPredicateBuilder.java

```diff
@@ -63,12 +66,44 @@ public void defaultOperator(BooleanOperator operator) {
 		}
 	}
 
+	@Override
+	public void enable(SimpleQueryFlag operation) {
+		if ( this.operations == null ) {
+			this.operations = EnumSet.noneOf( SimpleQueryFlag.class );
```

Comment:
If the function "disable" is used for the first time, it will set all to enabled and the indicated one to disable.

If the function "enable" is used for the first time, it will set all to disabled and the indicated one to enable.